plugins/invoice-csv-export/src/public.php

```diff
@@ -28,30 +31,29 @@
         'createdDateTo' => $_GET['until'],
     ];
 
-    $countries = $api->query('countries');
+    $countries = $api->get('countries');
     $states = array_merge(
         // Canada
-        $api->query('countries/54/states'),
+        $api->get('countries/54/states'),
```

Comment:
I don't like the magic numbers, can we somehow abstract this or create constants?

Reply:
It's available in https://ucrm.docs.apiary.io/#reference/general/countries/get
We could probably do that in the future, but here it's out of scope.